Makefile

```diff
@@ -32,6 +32,11 @@ test: check
 shellcheck:
 	./run-shellcheck.sh `git ls-files *.sh`
 
+pre-commit-check:
+	pip3 install pre-commit
```

Comment:
I'd rather not install anything this way. Imagine you'd run `make pre-commit-check` under the root and the command installs pre-commit and all its 12 dependencies into your system site-packages - this might be a problem in case of any conflict with your system. I'd honestly just assume that the command is available and let it show an error when it isn't.

Reply:
and what about if pre-commit does not exist in CI? This means, that pre-commit checker will not work.

Reply:
Yes, but you can install it in the same way you do it for shellcheck: https://github.com/sclorg/container-common-scripts/blob/82c7f58e9a293aaaa5fcf3f45ae51b7678646d28/.github/workflows/sanity-tests.yaml#L36